js/main.js

```diff
@@ -0,0 +1,163 @@
+var $searchedUl = document.querySelector('.searched-ul');
+var $searchedLi = document.querySelector('.searched-li');
+var $searchResultH2 = document.querySelector('.search-result-h2');
+
+$headerSearchIcon.addEventListener('click', showSearchPage);
+$homeFirstBtn.addEventListener('click', showSearchPage);
+$footerPlusIcon.addEventListener('click', showSearchPage);
+
+$homeSecontBtn.addEventListener('click', showAlbumPage);
+$footerAlbumIcon.addEventListener('click', showAlbumPage);
+
+$footerHomeIcon.addEventListener('click', showHomePage);
+
+$formSearchBar.addEventListener('submit', retrieveResult);
+
+function showSearchPage(event) {
+  $homeHeader.className = 'home-header hidden';
+  $homeH2andImg.className = 'home-h2-and-img hidden';
+  $homeTwoBtns.className = 'home-two-btns hidden';
+  $myPlace.className = 'my-place hidden';
+  $myPlaceUl.className = 'my-place-ul hidden';
+  $footerHomeIcon.className = 'footer-home-icon';
+  $footerAlbumIcon.className = 'footer-album-icon';
+
+  $searchHeader.className = 'search-header';
+  $searchH2andImg.className = 'search-h2-and-img';
+  $footerPlusIcon.className = 'footer-plus-icon on';
+}
+
+function showHomePage(event) {
+  $searchHeader.className = 'search-header hidden';
+  $searchH2andImg.className = 'search-h2-and-img hidden';
+  $myPlace.className = 'my-place hidden';
+  $myPlaceUl.className = 'my-place-ul hidden';
+  $searchResult.className = 'search-result hidden';
+  $searchedUl.className = 'searched-ul hidden';
+  $footerPlusIcon.className = 'footer-plus-icon';
+  $footerAlbumIcon.className = 'footer-album-icon';
+
+  $homeHeader.className = 'home-header';
+  $homeH2andImg.className = 'home-h2-and-img';
+  $homeTwoBtns.className = 'home-two-btns';
+  $footerHomeIcon.className = 'footer-home-icon on';
+}
+
+function showAlbumPage(event) {
+  $homeH2andImg.className = 'home-h2-and-img hidden';
+  $homeTwoBtns.className = 'home-two-btns hidden';
+  $searchHeader.className = 'search-header hidden';
+  $searchH2andImg.className = 'search-h2-and-img hidden';
+  $searchResult.className = 'search-result hidden';
+  $searchedUl.className = 'searched-ul hidden';
+  $footerHomeIcon.className = 'footer-home-icon';
+  $footerPlusIcon.className = 'footer-plus-icon';
+
+  $homeHeader.className = 'home-header';
+  $myPlace.className = 'my-place';
+  $myPlaceUl.className = 'my-place-ul';
+  $footerAlbumIcon.className = 'footer-album-icon on';
+}
+
+function searchResultPage(event) {
+  $homeHeader.className = 'home-header hidden';
+  $homeH2andImg.className = 'home-h2-and-img hidden';
+  $homeTwoBtns.className = 'home-two-btns hidden';
+  $searchH2andImg.className = 'search-h2-and-img hidden';
+  $myPlace.className = 'my-place hidden';
+  $myPlaceUl.className = 'my-place-ul hidden';
+  $footerHomeIcon.className = 'footer-home-icon';
+  $footerAlbumIcon.className = 'footer-album-icon';
+
+  $searchHeader.className = 'search-header';
+  $footerPlusIcon.className = 'footer-plus-icon on';
+}
```

Comment:
Instead of having an individual function for each view, you should add a `data-view` attribute to each of these elements stating the view it needs to go to, and get that value when a link is clicked. Then, loop over the list of available views, and check if that is the view mentioned. If it is, show that view. If it's not, hide that view. I recommend reviewing `javascript-view-swapping` for a refresher.